src/docs/components/variants/tag/SubtleStyle.tsx

```diff
@@ -0,0 +1,19 @@
+import Tag from '../../../../library/components/Tag';
+import Example from '../../../shared/components/Example';
+
+const SubtleStyle = (): JSX.Element => {
+  return (
+    <>
+      <Example>
+        <Tag variant="subtle">Subtle tag</Tag>
+      </Example>
+      <Example isDarkTheme>
```

Comment:
We're including `={true}` in other tag style variant files. Can we standardize how we're writing isDarkTheme across variant files? (Could be another tiny cleanup ticket)

Reply:
I thought we would've done that too, but the other files still have it in dev. Can we look into why that isn't getting caught locally and in CI/CD?